hawc/apps/assessment/serializers.py

```diff
@@ -21,8 +21,8 @@ class Meta:
 
 
 class AssessmentSerializer(serializers.ModelSerializer):
-    rob_name = serializers.CharField(source="get_rob_name_display")
-    dtxsids = DSSToxSerializer(many=True)
+    rob_name = serializers.CharField(source="get_rob_name_display", read_only=True)
+    dtxsids = DSSToxSerializer(many=True, read_only=True)
```

Comment:
does this mean that you can't add dtxsids? Could we do that trick where this one is read_only and then we have another `dtxsids_ids` which is write_only?

```python
dtxsids_ids = serializers.PrimaryKeyRelatedField(many=True, write_only=True)
```